ocrd_cis/ocropy/segment.py

```diff
@@ -555,6 +551,7 @@ def _process_element(self, element, ignore, image, coords, zoom=1.0, rogroup=Non
             report = check_region(element_bin, zoom)
             suffix = element.id + '.IMG-CLIP'
         self.logger.info(f'Computing line segmentation for {element_name} "{element.id}"')
+        element_name_id = f'{element_name} "{element.id}"'
```

Comment:
good idea!

Reply:
Yes, should be also easier to refactor in the future. String computations are still worth optimizing.